packages/typescript-runtime/bin/nuxt-ts.js

```diff
@@ -1,20 +1,11 @@
 #!/usr/bin/env node
 
-const path = require('path')
-const { resolveNuxtBin, getRootdirFromArgv, registerTSNode } = require('..')
+const cli = (() => { try { return require('@nuxt/cli') } catch { return require('@nuxt/cli-edge') } })()
```

Comment:
Optional catch binding doesn't work with node.js below version 10.3

Reply:
Nice catch @guAnsunyata, could you please consider opening a new issue ? Thanks !

Reply:
https://github.com/nuxt/typescript/commit/69f2d2053af8a00b4d11dc155a868183a0a392fb